x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/expression/function/EqlFunctionRegistry.java

```diff
@@ -42,6 +43,7 @@ public EqlFunctionRegistry() {
                 def(EndsWith.class, EndsWith::new, "endswith"),
                 def(IndexOf.class, IndexOf::new, "indexof"),
                 def(Length.class, Length::new, "length"),
+                def(Match.class, Match::new, "match", "matchlite"),
```

Comment:
`matchLite` as an alias to `match`?

Reply:
I originally had "matchLite" but apparently the aliases have to also be normalized to lowercase, so it's "matchlite"

both functions have been around for a while, but `matchLite` was more limited than regex -- had character clasess and `*`, `*?`,  and `+` because of our underlying implementation.

now, they both have the same functionality, so the alias is just for backwards compatibility.